praxis/layers/activations.py

```diff
@@ -67,7 +67,14 @@ def __call__(self, inputs: JTensor) -> JTensor:
     """Applies the activation function."""
     return jax.nn.relu(inputs)
 
+class sineReLU(BaseActivation):
```

Comment:
nit: do you mind capitalizing sineReLU following naming convention in the rest of the file?

Reply:
Sure! Doing this in a moment